home-page.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://shed-ui.netlify.app/component.css');
+@import url(/utils.css);
+
+.nav-icon-container{
+    margin-left: auto;
+}
+.top{
```

Comment:
You have made the same definition for top class in [utils](https://github.com/ShekharDhangar/shed-quiz/pull/1/files#diff-d8aa325440ae83b7598877130ebfad47f911f11ca7a8ffd03bc6b83de7bec1acR6) as well. One of these can be removed.

Reply:
Done